CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
 # Change Log
+## [1.2.21](https://github.com/ably/ably-java/tree/v1.2.21)
```

Comment:
Something is wrong with the state of this branch if it's showing changes made in a release PR. Looks like a merge operation has gone wrong at some point.

Reply:
It is fixed now.